timeless/restaurants/tables/views.py

```diff
@@ -0,0 +1,63 @@
+"""tables views module.
+@todo #112:30min Continue implementing list_tables(), create(), edit() and
+ delete() methods, using SQLAlchemy and Location model. In the index page it
+ should be possible to sort and filter for every column. Location management
+ page should be accessed by the Location page. Update html templates when
+ methods are implemented. Create more tests for all methods.
+ Table management pages should be accessed by the Location and Floor pages.
+"""
+import datetime
+
+from flask import (
+    Blueprint, flash, redirect, render_template, request, url_for
+)
+
+
+bp = Blueprint("table", __name__, url_prefix="/tables")
+
+
+@bp.route("/")
+def list():
+    # remove this dummy tables object and use db
+    tables = [{
+        "id": 1,
+        "name": "Test table",
+        "description": "This is a test",
+        "floor_id": 1,
+        "x": 10,
+        "y": 20,
+        "width": 12,
+        "height": 22,
+        "status": 1,
+        "max_capacity": 6,
+        "multiple": True,
+        "playstation": False,
+        "shape_id": 3,
+        "min_capacity": 4,
+        "deposit_hour": 3,
+        "created": datetime.datetime.now(),
+        "updated": datetime.datetime.now(),
+        }]
+    return render_template("restaurants/tables/list.html", tables=tables)
+
+
+@bp.route("/create", methods=("GET", "POST"))
+def create():
+    if request.method == "POST":
+        flash("Create not yet implemented")
+    action = "create"
+    return render_template("restaurants/tables/create_edit.html", action=action)
```

Comment:
@krzyk has the architecture idea changed? To my understanding, the backend should expose a RESTful API (most probably data presented in JSON) and the front-end should be a ReactJS client app -- so why are these methods rendering HTML templates?

Reply:
@amihaiemil I would link you to a comment in previous PR here: https://github.com/timelesslounge/timelessis/pull/82 But probably because of the repo change it gives 500.
Basically until we have the UI from the POs we will do html templates @emilianodellacasa 
You can also look at other views.py files.

Reply:
@amihaiemil not my decision :) -> @emilianodellacasa 

BTW. With html you can at least do management, with json, you would have a "curl" interface only :)

Reply:
@krzyk > with json, you would have a "curl" interface only :)

Or maybe Swagger, quite nice tool. But yes, the ARC's decision.

Reply:
@amihaiemil @krzyk The fact is that I think React is a little too much just for insert/edit/list/delete objects. I would like to pursue a hybrid approach:

1) jinja for simple management;
2) React for more complex pages (they are coming soon, don't worry);

Also, by my understanding ([here](https://stackoverflow.com/questions/32705233/how-to-use-jinja2-server-side-rendering-alongside-react-without-violating-inline) for example) it is possible to use React inside jinja